docs/resources/agent.md

```diff
@@ -97,4 +114,5 @@ Required:
 Optional:
 
 - `display_name` (String) The user-facing name of this value.
+- `order` (Number) The order determines the position of agent metadata in the UI/CLI presentation. The lowest order is shown first and parameters with equal order are sorted by key (ascending order).
```

Comment:
Should we not just disallow parameters with same order?

Reply:
I wouldn't complicate things here. Imagine you have 100 metadata blocks, 1, 2,... 100. Now, you want to insert 57. With a rule forbidding duplicates, you would have to rename the rest. In this case, you can leave the duplicate.